assets/i18n/qqq.json

```diff
@@ -363,6 +363,7 @@
 	"I18N_TOPNAV_NOTIFICATIONS": "Text displayed inside a drop-down menu. - The list is shown after the user hovers over their profile picture. When the option is clicked, the notifications page is loaded.\n{{Identical|Notification}}",
 	"I18N_TOPNAV_PREFERENCES": "Text displayed inside a drop-down menu. - The list is shown after the user hovers over their profile picture. When the option is clicked, the preference page is loaded where the user can modify the account information.\n{{Identical|Preferences}}",
 	"I18N_TOPNAV_SIGN_IN": "Text displayed in the top right corner of the page. - When clicked, the user is redirected to the sign in page where he can access his account.\n{{Identical|Sign in}}",
+	"I18N_TOPNAV_SIGN_IN_WITH_GOOGLE": "Text displayed inside the Sign In dropdown. - When clicked, the user is signed in with Google Outh",
```

Comment:
Oauth is misspelled.

Do we need to provide a character limit here (as info for translators)?